scripts/setup-ssh-keys.sh

```
@@ -0,0 +1,767 @@
+
+# Configuration (matching install.sh conventions)
+DOCKER_MCP_DIR="${HOME}/.docker-mcp"
+SSH_KEY_NAME="docker-mcp-key"
+SSH_KEY_PATH="${DOCKER_MCP_DIR}/ssh/${SSH_KEY_NAME}"
+CONFIG_DIR="${DOCKER_MCP_DIR}/config"
+DATA_DIR="${DOCKER_MCP_DIR}/data"
+
+# Additional settings
+SSH_CONFIG="${SSH_CONFIG:-${HOME}/.ssh/config}"
+PARALLEL_JOBS="${PARALLEL_JOBS:-10}"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.10.0)

[warning] SCRIPT_DIR appears unused. Verify use (or export if used externally).

(SC2034)